assets/css/tasks/_contact-tasks.css

```diff
@@ -0,0 +1,70 @@
+/* CONTACT PAGE */
+
+/* assets/css/pages/_contact.css */
+.contact-section {
+    background-image: url(/img/contact-img.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: right 20rem bottom 0;
+    
+}
+
+.form {
+    padding: 10px 10px;
+    width: 40%;
+    background-color: white;
+    border-radius: 1px;
+    position: relative;
+    left: 3px;
+    bottom: 1px;
+    text-align: left;
+    
+}
+
+.form-title {
+    font-weight: lighter
+    position: relative;
+    color:#676767;
+    
+    
+}
+
+.form-input {
+    display: 100%;
+    width: 80%;
+    height: 200px;
```

Comment:
This `height: 200px` is really big.  May be, Will be better with smaller, `50px` or `3rem`?